tutorials/hamiltonian_simulation/hamiltonian_simulation_guide.ipynb

```diff
@@ -0,0 +1,604 @@
+{
```

Comment:
<p>The notebook in the link shows two things:</p><ol><li>How to use these methods for Hamiltonian derived from molecules.</li><li>The optimization of these functions by the synthesis engine.</li></ol><p>I am not sure it gives more experience, it mainly highlights the latter (which is a good thing to mention).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>